docs/installing-on-gke.md

```diff
@@ -0,0 +1,321 @@
+The command deploys OSDFIR Infrastructure on the Kubernetes cluster while enabling
+Turbinia GCP integration.
+
+Verify the deployment:
+
+```bash
+kubectl get pods
+```
+
+You should see pods for Timesketch, Turbinia, and Yeti in a Running state. It may
+take a few minutes for all the services to deploy.
+
+### Provisioning shared filestorage (optional)
+
+While this example uses a single node, you might need persistent storage for
+multi-node deployments. To enable persistent storage with a `ReadWriteMany` PVC,
+add the following `--set` flags to your helm install command:
+
+```bash
+--set persistence.storageClass="standard-rwx" \
+--set persistence.accessModes[0]="ReadWriteMany"
+```
+
+This configures OSDFIR Infrastructure to provision a GCP Filestore instance with
+`ReadWriteMany` access mode, which is suitable for multi-node clusters where
+shared storage is required.
+
+## Step 5: Setup dfTimewolf and CLI configs
+
+OSDFIR Infrastructure utilizes dfTimewolf for orchestrating forensic collection
+and processing. dfTimewolf allows you to define "recipes" that specify how data
+should be collected, processed by tools like Turbinia, and exported to platforms
+like Timesketch.
+
+To install dfTimewolf, you'll need to have Python 3.11 or greater, `git`, and
+`pip` installed on your machine. dfTimewolf uses Poetry for simplified dependency
+management. Follow these steps to install dfTimewolf:
+
+```bash
+git clone https://github.com/log2timeline/dftimewolf.git && cd dftimewolf
+pip install poetry
+poetry install && poetry shell
+```
+
+Retrieve the Timesketch password from your deployment. For example, to grab
+it from a release named `my-release`, run:
+
+```bash
+kubectl get secret --namespace default my-release-timesketch-secret -o jsonpath="{.data.timesketch-user}" | base64 -d
+```
+
+dfTimewolf uses a configuration file called `.dftimewolfrc` to store settings
+such as your Timesketch credentials and endpoint. This allows you to avoid
+entering these details every time you run a recipe.
+
+Now, create a `.dftimewolfrc` file in your HOME directory, replacing
+`$TIMESKETCH_PASSWORD` with the Timesketch password retrieved in the previous step:
+
+```bash
+cat >> ~/.dftimewolfrc << EOF
+{
+"timesketch_username": "timesketch",
+"timesketch_password": "$TIMESKETCH_PASSWORD",
+"timesketch_endpoint": "http://127.0.0.1:5000",
+"turbinia_api": "http://127.0.0.1:8000"
+}
+EOF
+```
+
+Now you have dfTimewolf installed and configured to interact with your OSDFIR
+Infrastructure deployment.
+
+## Step 6: Process a Google Cloud Disk
+
+With OSDFIR Infrastructure deployed and dfTimewolf installed and configured,
+you're ready to process a GCP disk.
+
+This example uses the dfTimewolf `gcp_turbinia_ts` recipe, which processes an
+existing GCP persistent disk with Turbinia and sends the resulting Plaso timeline
+to Timesketch.
+
+First, create a disk to process using a name such as `test-disk`:
+
+```bash
+gcloud compute disks create test-disk --zone $ZONE
```

Comment:
This created an empty 500GB disk...maybe create a 10GB disk with the default debian12 image?

Reply:
Agreed, updated this part to include creating a debian image instead